pkg/internal/webhook/validator_test.go

```diff
@@ -134,6 +137,23 @@ func Test_validate(t *testing.T) {
 			webhooks:          []approver.Webhook{passingWebhook, failingWebhook},
 			wantsErr:          true,
 		},
+		"if a registered webhook does not allow CertificteRequestPolicy without further detail, return an error": {
+			crp: &policyapi.CertificateRequestPolicy{
+				TypeMeta:   testTypeMeta,
+				ObjectMeta: testObjectMeta,
+				Spec: policyapi.CertificateRequestPolicySpec{
+					Plugins: map[string]policyapi.CertificateRequestPolicyPluginData{"foo": {}, "bar": {}},
+					Selector: policyapi.CertificateRequestPolicySelector{
+						Namespace: &policyapi.CertificateRequestPolicySelectorNamespace{
+							MatchLabels: map[string]string{"foo": "bar"},
+						},
+					},
+				},
+			},
+			registeredPlugins: []string{"foo", "bar"},
+			webhooks:          []approver.Webhook{passingWebhook, notAllowedWebhookNoDetail},
+			wantsErr:          true,
```

Comment:
Since we are not comparing the error messages anymore, it is not clear whether this change caused the error messages to change. 😟